priv/templates/channel/channel.ex

```diff
@@ -0,0 +1,18 @@
+defmodule <%= module %>Channel do
+  use <%= base %>.Web, :channel
+
+  def join("<%= plural %>:lobby", _auth_message, socket) do
+    {:ok, socket}
+  end
+
+  def join("<%= plural %>:" <> _<%= singular %>_id, _auth_message, socket) do
+    {:ok, socket}
+  end
+
+  <%= for event <- events do %>
+    def handle_in("<%= event %>", attrs, socket) do
+      broadcast! socket, event, attrs
```

Comment:
This is tricky because not all events are going to be broadcasted. I'm not sure what the best default body would be here. We have three different scenarios. Sync reply, async push, broadcast + combination of first two